lib/muncher_api_wrapper.rb

```diff
@@ -0,0 +1,35 @@
+require 'httparty'
+class MuncherApiWrapper
+
+
+  class MuncherError < StandardError; end
+  def self.list_recipes(food)
+    token = ENV["MUNCHER_API_KEY"]
+    id = ENV["MUNCHER_API_ID"]
+
+    url = "https://api.edamam.com/search?q=#{food}&app_id=#{id}&app_key=#{token} &from=#{0}&to=#{1200}"
+    encoded_url = URI.encode(url)
+
+    response = HTTParty.get(encoded_url)
+
+    unless response["more"]
+      raise StandardError.new(response["error"])
+    end
```

Comment:
I like that you're checking for errors here. However, I'm not sure that Edamam sends back an error message under the `"error"` key. It's important to make sure your error handling matches what the errors look like.